docs/api/hardhat/hardhat-zksync-vyper.md

```diff
@@ -35,14 +33,11 @@ This plugin most often will not be used directly in the code.
 
 ```typescript
 zkvyper: {
-  version: "0.1.0",
+  version: "1.3.4",
```

Comment:
1.3.5

Reply:
fixed in [a973126](https://github.com/matter-labs/zksync-web-era-docs/pull/370/commits/a973126b16b5c1cd1e2d9b2440f779d097c854ce)